node/nanoexpress/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:12.6.0
+
+RUN npm -g install pm2
+
+WORKDIR /usr/src/app
+
+COPY app.js package.json ./
+
+RUN apk update && apk add --no-cache git
+RUN npm install
+RUN ln -s /lib/libc.musl-x86_64.so.1 /lib/ld-linux-x86-64.so.2
+
+ENV NODE_CLUSTER_SCHED_POLICY=rr
```

Comment:
what is this env var made for ?

Reply:
For able to using cluster-mode (aka PM2 clustering). This limitation referring to uWebSockets.js. Currently I don't know C++ good, so I leaved this bug, with workaround

Reply:
its specific to `uWebSocket` then ?

Reply:
Yes, currently there no CORE solution. In Docker (Linux) maybe works, maybe not at default. So I decided pass `ENV` which prefers able to use Multi-core for better handling/performance.